utils_cv/detection/plot.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Helper module for visualizations
+"""
+from typing import List, Union, Tuple, Callable, Any
+from pathlib import Path
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+from PIL.JpegImagePlugin import JpegImageFile
+
+from .references.coco_eval import CocoEvaluator
+
+
+class BoundingBoxParams:
```

Comment:
Name does not make clear that these params have to do with plotting. Maybe use PlotSettings?